test/performance/README.md

```diff
@@ -0,0 +1,25 @@
+# Profiles Performance Test
+
+The `profiles.jmx` [JMeter](http://jmeter.apache.org/) test will call the Profiles application for a number of different surgeries.
+Several parameters are exposed to customise the base test.
+
+
+| Parameter                         | Description                                                        | Default               |
+|:---------------------------------|:-------------------------------------------------------------------|-----------------------|
+| `hostname`                       | URL of server to test                                              | profiles.dev.beta.nhschoices.net|
```

Comment:
Usually, performance tests are done on the staging environment as it's deemed closer to the production configuration. In our case, it should have multiple images per app and theoretically, that should take the load.